pkgs/by-name/de/degit-rs/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "degit-rs";
+  version = "0.1.3-unstable";
+
+  src = fetchFromGitHub {
+    owner = "psnszsn";
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21